src/index.tsx

```diff
@@ -157,8 +157,10 @@ const launchPaymentsUI = (
         error?: CallbackStatus,
       ) => {
         if (error) {
-          console.error(`payments error: ${JSON.stringify(error)}`);
-          reject(error);
+          console.info(
```

Comment:
is there anything "interesting" in the `data` that worth printing to console?

Reply:
In my opinion no, we can remove this print as it contains internal implementation data

Reply:
@kostat Do you want to remove the console, or add to it?